app/frontend/src/search/ui/alert.js

```diff
@@ -1,43 +1,49 @@
+/* eslint-disable max-len */
```

Comment:
I'm uncomfortable with disabling rules at all, let alone for entire files.
Applies to 4 files.

Reply:
Sorry, should have read the PR description!

Reply:
personally i disagree, it depends on the rule. in this case it is indicating that the code might be hard to read because of the length of the line. but when you are talking about strings being on one line refactoring them into smaller lines makes them harder to read and in the case of tests can break your tests. if the rule was === rule for instance, i would never disable this as this indicates your code more than likely has a problem. ill meet you halfway and disable the lines explicitly that it applies to rather than the whole file

Reply:
but you are absolutely right picking up anything lint wise that says disable, its more often than not a code smell if it hasn't been thought about